src/rqt_launch/launch_widget.py

```diff
@@ -237,14 +237,14 @@ def _refresh_launchfiles(self, package=None):
         if package is None or len(package) == 0:
             return
         self._launchfile_instances = []  # Launch[]
-        #TODO: RqtRoscommUtil.list_files's 2nd arg 'subdir' should NOT be
+        # TODO: RqtRoscommUtil.list_files's 2nd arg 'subdir' should NOT be
         # hardcoded later.
         _launch_instance_list = RqtRoscommUtil.list_files(package,
-                                                         'launch')
+                                                          'launch')
 
         rospy.logdebug('_refresh_launches package={} instance_list={}'.format(
-                                                       package,
-                                                       _launch_instance_list))
+            package,
```

Comment:
Align with str.format

Reply:
This alignment is actually fine according to PEP 8: https://www.python.org/dev/peps/pep-0008/#indentation

Reply:
It may be fine, but it doesn't make it more readable

Reply:
That's a highly subjective claim. I actually think it's more readable :smile:

Reply:
Turns out the entire thing is less than 120 char so I'm making it one line. That way no one is happy :stuck_out_tongue_closed_eyes: